intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -1194,6 +1195,18 @@ \subsection{Explicit version numbers}
 latter version, rather than the intended one, which may lead to unexpected
 problems.
 
+To list the available versions, you can use the \verb|module avail| command. For
+example, to list the available versions of the GSL library:
+
+\begin{prompt}
+%\shellcmd{module avail GSL}%
+FGSL/1.1.0-intel-2016b (D)    GSL/2.3-intel-2016b
+GSL/2.1-intel-2016b           GSL/2.3-intel-2017a
+GSL/2.2.1-intel-2016b         GSL/2.4-GCCcore-6.4.0 (D)
+GSL/2.3-foss-2016b
+
+\end{prompt}
```

Comment:
Why is this simply not a reference to the section about `module avail` we have already?

Also `GSL` is not the best example, since `FGSL` also pops up. At the very least explain this?

I'd also use `module avail GSL/`, to avoid partial match on anything module that includes `GSL` (that's particularly painful with `module avail R` if module is configured to be case-insensitive (which is site-specific))...

Reply:
Because our current section about `module avail` doesn't talk about listing all versions: it only explains the `module avail` command that shows all modules.  Our Linux manual has a chapter (of one page) about the `module` command. Maybe that chapter should be moved to the HPC manual? (here's a link to it http://hpcugent.github.io/vsc_user_docs/pdf/intro-Linux-linux-gent.pdf#chapter.6)

Reply:
I think it makes more sense to explain the `module` command in the HPC tutorial, since it's quite specific to HPC. See also https://github.com/hpcugent/vsc_user_docs/issues/192